.github/workflows/release.yaml

```diff
@@ -16,6 +16,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Fetch tags
+        run: git fetch --force --tags
```

Comment:
Good catch! But it seems this is not needed. Have you tried without it?

```suggestion
```

References:

- https://github.com/actions/checkout/blob/b4ffde65f46336ab88eb53be808477a3936bae11/README.md?plain=1#L7
- https://github.com/goreleaser/goreleaser-action/blob/e0e636a9515af0b3b3fa2d5c5fa440e36d49812b/README.md?plain=1#L77

Reply:
Done!. I have removed it